pyscriptjs/tests/integration/test_py_runtime_config.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
the presence of this file is wrong and it's probably a merge artifact.
It was renamed into `test_py_config.py` by PR 806, see here:
https://github.com/pyscript/pyscript/pull/806/files#diff-be855b518404836a1b9ed7ccecb56a8055aa1b0a11f3c73743cf44ac18a633bb

You can safely remove it.